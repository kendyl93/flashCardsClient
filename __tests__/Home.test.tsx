/**
 * @format
 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreen from '../src/Views/Home';

describe('HomeScreen', () => {
  it('renders correctly', async () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
