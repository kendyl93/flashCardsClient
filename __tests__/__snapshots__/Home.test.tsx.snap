// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "whitesmoke",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello React Native Navigation ðŸ‘‹
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
            Object {
              "color": "#710ce3",
            },
          ]
        }
      >
        Push Settings Screen
      </Text>
    </View>
  </View>
</View>
`;
